guides/source/api_app.md

```diff
@@ -332,6 +332,28 @@ will be:
 { :person => { :firstName => "Yehuda", :lastName => "Katz" } }
 ```
 
+### Using Session Middlewares
+The following middlewares are (by default) included for session management.  If one of your API clients is a browser, you might want to add one of these back in:
+- `ActionDispatch::Session::CacheStore`
+- `ActionDispatch::Session::CookieStore`
+- `ActionDispatch::Session::MemCacheStore`
+
+The trick to adding these back in is that, by default, they are passed `session_options`
+when added (including the session key), so you can't just add a `session_store.rb` initializer, add
```

Comment:
I see how this is confusing.  IIRC, the point of this comment is not to do with where you add the middleware (which should be in `application.rb`), but rather that the conventional `session_store.rb` initializer won't work if you are adding these middlewares in manually, because you have to pass session_options to these middlewares, and they won't have been set in the correct scope yet if you use the conventional session_store.rb initializer to set them.  Therefore you have to manually set them in application.rb before "using" the middleware, so that you can pass them to the middleware.